[Feature]: Adds admin authentication functionalities

sharthak1095 opened the pull request:
Adding full passport authentication and authorization system for admin user

sharthak1095 committed:
[Feature]: Adds admin login functionality

sharthak1095 committed:
[Feature]: Adds admin user registration process

sayan95 commented:
one bug has been found.

Access token assignment workflow should change.
Do not need to rely on access_token cookie.

sharthak1095 committed:
[Bugfix]: resolves access token assignment issue

sharthak1095 committed:
[Feature]: Adds admin profile activities
1. Adds profile view activity
2. Adds profile info update activity
3. Adds profile password change activity

sharthak1095 committed:
[Feature]: Adds admin logout functionality

sayan95 commented:
The feature passport authentication functionality has been implemented completely. Thus closing off this request.
